spec/acts_as_tag_pgarray/taggable_spec.rb

```diff
@@ -1,162 +1,275 @@
 require "spec_helper"
 
 describe ActsAsTaggableArrayOn::Taggable do
-  before do
-    @user1 = User.create name: "Tom", colors: ["red", "blue"], sizes: ["medium", "large"], codes: [456, 789]
-    @user2 = User.create name: "Ken", colors: ["black", "white", "red"], sizes: ["small", "large"], codes: [123, 789]
-    @user3 = User.create name: "Joe", colors: ["black", "blue"], sizes: ["small", "medium", "large"], codes: [123, 456, 789]
-    @admin1 = Admin.create name: "Dick", colors: ["purple", "orange"], sizes: ["medium", "large"], codes: [123, 456, 789]
-    @admin2 = Admin.create name: "Harry", colors: ["white", "blue"], sizes: ["small", "large"], codes: [456, 123]
-
-    User.acts_as_taggable_array_on :colors
-    User.acts_as_taggable_array_on :sizes
-    User.taggable_array :codes
-
-  end
-
   context "without database table" do
     it "doesn't fail on class method call" do
       class Dummy < ActiveRecord::Base; end
       Dummy.acts_as_taggable_array_on :tags
     end
   end
 
-  describe "#acts_as_taggable_array_on" do
-    it "defines named scope to match any tags" do
-      expect(User).to respond_to(:with_any_colors)
-    end
-    it "defines named scope to match all tags" do
-      expect(User).to respond_to(:with_all_colors)
-    end
-    it "defines named scope not to match any tags" do
-      expect(User).to respond_to(:without_any_colors)
-    end
-    it "defines named scope not to match all tags" do
-      expect(User).to respond_to(:without_all_colors)
-    end
-  end
+  context "without allowed option" do
```

Comment:
Can you please remove this context? I don't see any need for this to be in a root level context for testsuite.

Reply:
Did you read my [previous comment](https://github.com/tmiyamon/acts-as-taggable-array-on/pull/40#discussion_r908326474) about this? What do you think about moving all validations tests scenarios to another file?